packages/csharp/PortingAssistant/PortingAssistant.Api/Application.cs

```diff
@@ -109,6 +110,14 @@ public void SetupConnection(bool console = false)
                     };
                 }
             });
+
+            _connection.On<string, string> ("getAssessmentLog", request => {
+                if (!DateTime.TryParse(request, out DateTime date)) {
+                  date = DateTime.Today;
+                }
+                string dateString = date.ToString("yyyyMMdd");
+                return Path.Combine(_logDirectory, $"portingAssistant-assessment-{dateString}.log");
```

Comment:
you do not need to move this function in backend, electron side should also be able to resolve it.

Reply:
The log file folder is passed as parameter from electron side

Reply:
moved from backend to electron in latest changes